src/snps/io/writer.py

```diff
@@ -123,12 +123,14 @@ def _write_csv(self):
             filename = f"{clean_str(self._snps.source)}_{self._snps.assembly}{ext}"
 
         comment = (
-            f"# Source(s): {self._snps.source}\n"
-            f"# Build: {self._snps.build}\n"
-            f"# Build Detected: { self._snps.build_detected}\n"
-            f"# Phased: {self._snps.phased}\n"
+            f"# Pipeline Version: {self._kwargs.pop('pipeline_version')}\n"
+            f"# Source: SelfDecode\n"
+            f"# Assembly: GRCh{self._snps.build}\n"
+            # f"# Build Detected: { self._snps.build_detected}\n"
+            # f"# Phased: {self._snps.phased}\n"
             f"# SNPs: {self._snps.count}\n"
             f"# Chromosomes: {self._snps.chromosomes_summary}\n"
```

Comment:
please replace this line with:
`f"# Chromosomes: {self._snps.chromosomes_summary_output}\n"`
and add the new function **chromosomes_summary_output** in snps.snps

Reply:
@Titorat this is for you